addon/ops/export_vmf.py

```diff
@@ -52,9 +52,14 @@ def invoke(self, context, event):
         geometry_scale = props.geometry_scale
         texture_scale = props.texture_scale
         lightmap_scale = props.lightmap_scale
+        allow_skewed_textures = props.allow_skewed_textures
         align_to_grid = props.align_to_grid
 
-        settings = Settings(brush_objects, disp_objects, geometry_scale, texture_scale, lightmap_scale, align_to_grid)
+        settings = Settings(
+            brush_objects, disp_objects,
+            geometry_scale, texture_scale, lightmap_scale,
+            allow_skewed_textures, align_to_grid
+        )
```

Comment:
Kind of a nitpick, but I'd prefer a trailing comma here, so each item gets its own line.